### PR TITLE
Avoid using non-forking subshells with `-c` option

### PR DESCRIPTION
**Pull request by siteshwar**
Resolves: #9

### CODE REVIEW

**Comment by krader1961** [comment]
LGTM and I agree that disabling the problematic no-fork behavior for one-liner `ksh -c` invocations is a reasonable thing to do.

**Comment by krader1961** [comment]
For the record I don't understand what the code being removed was intended to do. It seems to me the fact the statements were specified via the `-c` flag rather than in a file (or stdin) should be completely irrelevant as to whether non-forking behavior was in effect. The usual logic for executing a "pipeline" of statements should handle whether non-forking behavior is used regardless of how the pipeline was given to the shell. Especially since the `-c` arg may be arbitrarily complex.

In other words, to state the blindingly obvious, whether the `echo x 1<>; a` is in a shell script or run via `ksh -c` should not matter.
